plugins/catalog/src/js/repositories/components/RepositoriesTable.js

```diff
@@ -1,69 +1,42 @@
 import classNames from "classnames";
-import { Confirm, Table } from "reactjs-components";
-import mixin from "reactjs-mixin";
+import { Table } from "reactjs-components";
 import PropTypes from "prop-types";
 /* eslint-disable no-unused-vars */
 import React from "react";
 /* eslint-enable no-unused-vars */
-import { StoreMixin } from "mesosphere-shared-reactjs";
-
-import CollapsingString from "./CollapsingString";
-import Config from "../config/Config";
-import CosmosPackagesStore from "../stores/CosmosPackagesStore";
-import List from "../structs/List";
-import ModalHeading from "./modals/ModalHeading";
-import ResourceTableUtil from "../utils/ResourceTableUtil";
+
+import CollapsingString from "#SRC/js/components/CollapsingString";
+
+import List from "#SRC/js/structs/List";
+import ResourceTableUtil from "#SRC/js/utils/ResourceTableUtil";
 import RepositoriesTableHeaderLabels
-  from "../constants/RepositoriesTableHeaderLabels";
-import StringUtil from "../utils/StringUtil";
-import TableUtil from "../utils/TableUtil";
-import UserActions from "../constants/UserActions";
+  from "#SRC/js/constants/RepositoriesTableHeaderLabels";
+import StringUtil from "#SRC/js/utils/StringUtil";
+import TableUtil from "#SRC/js/utils/TableUtil";
+import UserActions from "#SRC/js/constants/UserActions";
+import RepositoriesDelete from "../RepositoriesDelete";
 
 const METHODS_TO_BIND = [
   "getHeadline",
   "getPriority",
   "getRemoveButton",
   "handleDeleteCancel",
-  "handleDeleteRepository",
   "handleOpenConfirm"
 ];
 
-class RepositoriesTable extends mixin(StoreMixin) {
+class RepositoriesTableUI extends React.Component {
```

Comment:
The name of the file is `RepositoriesTable.js` any particular reason why did you go with `RepositoriesTableUI`?

Reply:
No, actually I have moved all the extracted UI components to UI, but decided to revert that in the end and forgot this one. Nice spot :D.